core/tests/performance_tests/base.py

```diff
@@ -36,9 +40,22 @@ def _record_page_metrics_from_cached_session(self, page_url):
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
     def _record_page_timings_for_url(self, page_url):
-        self.page_metrics = (
-            self.data_fetcher.get_page_timings_for_url(page_url))
+        page_session_metrics = []
+
+        for _ in range(self.SESSION_COUNT):
+            page_session_metrics.append(
+                self.data_fetcher.get_page_timings_for_url(page_url))
+
+        self.page_metrics = perf_domain.MultiplePageSessionMetrics(
+            page_session_metrics)
 
     def _record_page_timings_from_cached_session(self, page_url):
```

Comment:
Having the ability to customize the session_count in tests seems like a good idea. Furthermore, it reinforces your suggestion to include "average" in the 2 method names since it really is taking the average of a specified number of sampled sessions (even though in most cases this is default and hidden from the test).